tests/unit/s2n_ecc_preferences_test.c

```diff
@@ -45,5 +60,24 @@ int main(int argc, char **argv)
         EXPECT_FALSE(s2n_ecc_preferences_includes_curve(&s2n_ecc_preferences_20201021, TLS_EC_CURVE_ECDH_X25519));
     };
 
+    /* Test: check all security policies don't contain the placeholder curve "s2n_ecc_curve_none" */
+    {
+        const struct s2n_security_policy *security_policy = NULL;
+        const struct s2n_ecc_preferences *ecc_preferences = NULL;
```

Comment:
Why are you declaring these outside of the scope they're used in? Minimally that makes their use less clear, and can lead to bugs.